deeplake/util/transform.py

```diff
@@ -244,7 +243,11 @@ def _transform_and_append_data_slice(
 
         # failure at chunk_engine
         # retry one sample at a time
-        except SampleAppendError:
```

Comment:
If we wanted to catch Transform error, isn't it better to do like this?

```
except (TransformError, SampleAppendError):
    # TransformError pass through
    raise e
```

Reply:
we don't want to only capture TransformError and SampleAppendError, we want to capture all exceptions.